packages/react-rich-text-renderer/package.json

```diff
@@ -9,6 +9,7 @@
   "author": "Webiny Ltd",
   "license": "MIT",
   "dependencies": {
+    "@babel/plugin-transform-modules-commonjs": "^7.16.8",
```

Comment:
@endymion1818 this should be in `devDependencies` because it's only relevant for us, at publish time. Once the package is published, users don't need this in their projects. It's not a dependency for the core package logic.